Gruntfile.js

```diff
@@ -8,7 +8,8 @@ module.exports = function (grunt) {
         paths: {
             src: {
                 app: {
-                    smsapp: 'src/smsapp.js'
+                    smsapp: 'src/smsapp.js',
+                    ussd_registration: 'src/ussd_registration.js'
```

Comment:
The USSD line is shared for registration and quizzes so lets just make this `ussdapp` and file names etc.